--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   source/trakr_rl/trakr_rl/tasks/locomotion/mdp/rewards.py

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   scripts/rsl_rl/train.py
 


--- git diff ---
diff --git a/scripts/rsl_rl/train.py b/scripts/rsl_rl/train.py
index 39a34a9..8c19073 100644
--- a/scripts/rsl_rl/train.py
+++ b/scripts/rsl_rl/train.py
@@ -103,6 +103,7 @@ from isaaclab.envs import (
     ManagerBasedRLEnvCfg,
     multi_agent_to_single_agent,
 )
+from isaaclab.utils.assets import retrieve_file_path
 from isaaclab.utils.dict import print_dict
 from isaaclab.utils.io import dump_yaml
 from isaaclab_rl.rsl_rl import RslRlOnPolicyRunnerCfg, RslRlVecEnvWrapper
@@ -164,7 +165,10 @@ def main(env_cfg: ManagerBasedRLEnvCfg | DirectRLEnvCfg | DirectMARLEnvCfg, agen
 
     # save resume path before creating a new log_dir
     if agent_cfg.resume or agent_cfg.algorithm.class_name == "Distillation":
-        resume_path = get_checkpoint_path(log_root_path, agent_cfg.load_run, agent_cfg.load_checkpoint)
+        if args_cli.checkpoint:
+            resume_path = retrieve_file_path(args_cli.checkpoint)
+        else:
+            resume_path = get_checkpoint_path(log_root_path, agent_cfg.load_run, agent_cfg.load_checkpoint)
 
     # wrap for video recording
     if args_cli.video:
diff --git a/source/trakr_rl/trakr_rl/tasks/locomotion/mdp/rewards.py b/source/trakr_rl/trakr_rl/tasks/locomotion/mdp/rewards.py
index abc523d..0503abe 100644
--- a/source/trakr_rl/trakr_rl/tasks/locomotion/mdp/rewards.py
+++ b/source/trakr_rl/trakr_rl/tasks/locomotion/mdp/rewards.py
@@ -25,7 +25,7 @@ def energy(env: ManagerBasedRLEnv, asset_cfg: SceneEntityCfg = SceneEntityCfg("r
 
     qvel = asset.data.joint_vel[:, asset_cfg.joint_ids]
     qfrc = asset.data.applied_torque[:, asset_cfg.joint_ids]
-    return torch.sum(torch.abs(qvel) * torch.abs(qfrc), + torch.pow(torch.abs(qfrc), 0.0002), dim=-1) # penalizing high torques more to encourage smoother motions --> given in the trakr_legged_rl repository
+    return torch.sum(torch.abs(qvel) * torch.abs(qfrc) + torch.pow(torch.abs(qfrc), 0.0002), dim=-1) # penalizing high torques more to encourage smoother motions --> given in the trakr_legged_rl repository
 
 
 def stand_still(